### PR TITLE
Add a setting to enable mod manager and disable it by default

### PR DESCRIPTION
**Pull request by red-001**
Currently mod manager seems to be broken.

### CODE REVIEW

**Comment by est31** [comment]
The defaults for settings are managed in `defaultsettings.cpp`

File: minetest.conf.example [hunk 1]
```diff
@@ -603,4 +603,5 @@
 #    Comma-separated list of trusted mods that are allowed to access insecure
 #    functions even when mod security is on (via request_insecure_environment()).
 #secure.trusted_mods =
-
+#	enable mod manager
```
Comment (rubenwardy) [inline minetest.conf.example]: This is the wrong place, it's a client setting. Put it next to the other main menu settings (such as mainmenu_last_idx or something)

File: src/defaultsettings.cpp [hunk 2]
```diff
@@ -224,6 +225,8 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("font_size", fontsize.str());
 	settings->setDefault("mono_font_size", fontsize.str());
 #endif
+	//Main menu
+	settings->setDefault("enable_modmanager", "false");
```
Comment (rubenwardy) [inline src/defaultsettings.cpp]: Same here. This is the wrong place, it's a client setting. Put it next to the other main menu settings (such as mainmenu_last_idx or something)

**Comment by est31** [comment]
Not needed anymore, since d565ce95395ec514d48b86910ae333bd5bfaa951
I don't think we need a setting for this.

**Pull request closed without merging**